fpga_interchange/chip_info.py

```diff
@@ -670,7 +684,8 @@ def __init__(self):
         self.name = ''
         self.generator = ''
 
-        self.version = 0
+        # Note: Bump this whenever schema changes.
+        self.version = 1
```

Comment:
What do you mean by bumping exactly? (e.g. increasing by 1, assigning the version of the schema, etc.)

Reply:
Clarified comments.